tests/ut/py/test_worker/test_comm_provider_control.py

```
@@ -0,0 +1,512 @@
+
+def test_allocate_allocation_error_carries_provisional_id_and_zero_sections():
+    buf = bytearray(ALLOCATE_REPLY_BYTES)
+    error = RegionAllocationError(
+        provisional_resource_id=7,
+        control_kind=RegionControlErrorKind.BACKEND_FAILURE,
+        failed_part=RegionPartKind.COUNTER,
+        failed_operation=RegionOperationKind.ZERO_BYTES,
+        cleanup_debt_remaining=True,
+    )
+    encode_allocate_allocation_error_reply(buf, error)
+    tag, result, payload, counter, decoded = decode_allocate_reply(buf)
```

Ruff (0.16.1)

[warning] Unpacked variable `payload` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)